### ISSUE TITLE
No training

### ISSUE THREAD

**Issue by rawmean** [sorenbouma/keras-oneshot#3]
Did you really mean to disable training in the code?  The following loop never executes because the range is empty.

    for i in range(1,0):  #<-- this is an empty range
        (inputs,targets)=loader.get_batch(batch_size)
        loss=siamese_net.train_on_batch(inputs,targets)
        if i % evaluate_every == 0:
            val_acc = loader.test_oneshot(siamese_net,N_way,n_val,verbose=True)
            if val_acc >= best:
                print("saving")
                siamese_net.save('/home/soren/keras-oneshot/weights')
                best=val_acc

**Comment by sorenbouma** [comment]
My bad, I must have been experimenting with something and accidentally commited changes. Should be fixed now
